package.json

```diff
@@ -67,6 +67,7 @@
     "@bigbinary/eslint-plugin-neeto": "^1.0.26",
     "@bigbinary/neeto-commons-frontend": "^2.0.42",
     "@bigbinary/neeto-icons": "^1.9.22",
+    "@bigbinary/neeto-molecules": "^1.0.20",
```

Comment:
Why do we need neeto-molecules here? @KrishnapriyaSkk _A

Reply:
This was added to display the removed components in one of the removed stories, as discussed, I will remove that particular Table story inside layouts.